docs/EVG.md

```diff
@@ -1864,3 +1872,20 @@ guidelines:
    jurisdiction as specified in Section 9.2.4.  The stated address of the organisation 
    combined with the organization name SHALL NOT be the only information used to 
    disambiguate the organisation.
+   
+**LEI**: The Legal Entity Identifier (LEI) as specified in the ISO 17442 and registered
+   in the Global LEI Index.  
+   
+   1. This information SHALL be validated by matching the organization name and 
+   registration number found in the Global LEI Index against the Subject Organization 
+   Name Field (see 9.2.1) and Subject Registration Number Field (see 9.2.5) within the 
+   context of the subject’s jurisdiction as specified in Section 9.2.4.  
+   
+   2. The address information SHALL be compared in order to detect potential 
+   matching errors or errors in the registration information.  If errors are found, 
+   they SHALL be reported to the GLEIF Foundation and/or relevant registration 
+   authority.  
```

Comment:
I'm actually not a fan of this. I think, if anything, any matching errors should PREVENT the CA from issuing such a certificate.